NOTICE-binary

```diff
@@ -560,7 +560,7 @@ The Apache Software Foundation (http://www.apache.org/).
 
 
 Maven Artifact
-Copyright 2001-2019 The Apache Software Foundation
```

Comment:
If we update that date, should we use 2024?

Reply:
@mimaison, Copyright year updated for maven section